bee-message/src/message.rs

```diff
@@ -0,0 +1,242 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    parents::{ParentsBlock, ParentsType},
+    payload::Payload,
+    MessageId, MessagePackError, MessageUnpackError, ValidationError,
+};
+
+use bee_packable::{PackError, Packable, Packer, UnpackError, Unpacker};
+
+use crypto::{
+    hashes::{blake2b::Blake2b256, Digest},
+    signatures::ed25519,
+};
+
+use alloc::vec::Vec;
+use core::{convert::TryInto, ops::RangeInclusive};
+
+/// Range (in bytes) of a valid message length.
+pub const MESSAGE_LENGTH_RANGE: RangeInclusive<usize> = 53..=32768;
```

Comment:
A link to some spec explaining this odd number 53 would be helpful (speaking as someone who's trying to understand this crate)

Reply:
Hmm, I think this is a holdover from the previous version, it's different now. I'll update it.